python/metatensor-torch/tests/atomistic/model.py

```diff
@@ -60,7 +60,7 @@ def model():
         atomic_types=[1, 2, 3],
         interaction_range=4.3,
         outputs={
-            "tests::dummy::long": ModelOutput(
+            "energy": ModelOutput(
```

Comment:
this was intentionally using a non-standard output, to test these =). Could you create a separate model with different metadata instead?